template_rpm/09_cleanup.sh

```diff
@@ -49,4 +51,17 @@ fi
 
 truncate --no-create --size=0 "${INSTALL_DIR}"/var/log/dnf.*
 
+if containsFlavor selinux; then
+    sed -i -- 's/^SELINUX=\(disabled\|enforcing\)/SELINUX=permissive/' "$INSTALL_DIR/etc/selinux/config"
+    unshare --mount -- chroot -- "$INSTALL_DIR" /bin/sh -euc 'mount --bind -- / "$2"
+        umask 0755
+        mkdir -p -m 0700 -- /dev /var /run
+        mkdir -p -m 1777 -- /tmp /var/tmp /dev/shm
+        find /tmp /var/tmp /run /dev/shm -mindepth 1 -delete
+        : > /.qubes-relabeled
+        rm -f /.autorelabel
+        setfiles -r "$2" -- "/etc/selinux/$1/contexts/files/file_contexts" "$2"' sh targeted /mnt
+    echo 'selinux=1' > ./template.conf
```

Comment:
Ok, with the two PRs linked to https://github.com/QubesOS/qubes-issues/issues/7988, you can now do:
```
echo 'selinux=1' > "$TEMPLATE_CONF"
```